static/css/style.css

```diff
@@ -522,6 +526,68 @@ form.distribute-teams-form, form.clear-teams-form {
     width: 100%;
 }
 
+/* Showcase */
+.carousel.slide {
+    max-width: 800px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.carousel-item a img {
+    max-height: 500px;
+    min-width: auto;
+}
+
+.carousel-control-next:hover {
+    background: linear-gradient(270deg, rgba(54,54,54,1) 0%, rgba(122,122,122,0.5270483193277311) 48%, rgba(255,255,255,0) 100%);
```

Comment:
Whoa! That's a very specific opacity value! Can't we just use `rgba(122, 122, 122, 0.5)` and be happy?

Reply:
Yes, sorry. Just used an online generator to do that and never changed it haha.

Reply:
Changed as suggested.